serde/src/test/kotlin/com/amazon/ionhiveserde/objectinspectors/IonStructToMapObjectInspectorTest.kt

```diff
@@ -186,8 +186,8 @@ class IonStructToMapObjectInspectorTest {
         val tsList = listOf<Timestamp>(
                 Timestamp.valueOf("2014-10-14 12:34:56.789"),
                 Timestamp.valueOf("2015-10-16 12:34:56.789"))
-        assertEquals(ION.newInt(1), timestampIntSubject.getMapValueElement(timestampIntStruct, ION.newSymbol(tsList[0].toString())))
-        assertEquals(ION.newInt(2), timestampIntSubject.getMapValueElement(timestampIntStruct, ION.newSymbol(tsList[1].toString())))
+        assertEquals(ION.newInt(1), timestampIntSubject.getMapValueElement(timestampIntStruct, ION.newSymbol(getTsKey(tsList[0]))))
+        assertEquals(ION.newInt(2), timestampIntSubject.getMapValueElement(timestampIntStruct, ION.newSymbol(getTsKey(tsList[1]))))
```

Comment:
Let's just create the `com.amazon.ion.Timestamp` instances directly, using `com.amazon.ion.Timestamp.valueOf()`.

```
        val tsList = listOf<com.amazon.ion.Timestamp>(
                com.amazon.ion.Timestamp.valueOf("2014-10-14T12:34:56.789Z"),
                com.amazon.ion.Timestamp.valueOf("2015-10-16T12:34:56.789Z"))
```

Then there's no need for the `getTsKey` method.

Reply:
I'm using the forSqlTimestampZ to get around time zone issues due to the possibility of development machines running on non UTC time. The conversion to sql timestamp adds the proper offset for the time zone of the machine running the tests.

Reply:
Why do we want the offset to change depending on what machine is running the tests?

Reply:
java.sql.timestamp will always add the local time zone during conversion of the timestamps. For example, creating a struct using:

```
val tsList = listOf<com.amazon.ion.Timestamp>(
                com.amazon.ion.Timestamp.valueOf("2014-10-14T12:34:56.789Z"),
                com.amazon.ion.Timestamp.valueOf("2015-10-16T12:34:56.789Z"))
```
when deserialized as sql timestamps returns:

2014-10-14 05:34:56.789
2015-10-16 05:34:56.789